code/datums/keybinding/xeno.dm

```diff
@@ -191,6 +191,12 @@
 	description = "Prompt a wheel to choose which hugger you will deploy with the Throw Hugger ability."
 	keybind_signal = COMSIG_XENOABILITY_CHOOSE_HUGGER
 
+/datum/keybinding/xeno/drop_all_hugger
+	name = "drop_all_hugger"
+	full_name = "Carrier: Drop All Facehugger"
```

Comment:
```suggestion
	full_name = "Carrier: Drop All Facehuggers"
```

Reply:
uhh sure I guess since it's the only ability that can do multiple at once